services/ui-src/src/components/overlays/EntityDetailsOverlay.tsx

```diff
@@ -55,6 +56,11 @@ export const EntityDetailsOverlay = ({
   const { full_name, state } = useStore().user ?? {};
   const { updateReport } = useContext(ReportContext);
   const [spinner, setSpinner] = useState<Boolean>();
+  const isSAR = report?.reportType === ReportType.SAR;
```

Comment:
this almost seems worth putting in context with how much we use it, but not necessary

Reply:
I'ma make a ticket to refactor that into context. 👍🏻